libpod/runtime_linux.go

```diff
@@ -0,0 +1,43 @@
+//go:build !remote
+
+package libpod
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+// Print a warning if a path is not on a tmpfs.
+func warnIfNotTmpfs(paths []string) {
+	envVar := "PODMAN_SUPPRESS_TMPFS_WARNING"
+	if os.Getenv(envVar) != "" {
+		return
+	}
+
+	notTmpfs := make([]string, 0, len(paths))
+
+	for _, path := range paths {
+		statfs := new(unix.Statfs_t)
+		if err := unix.Statfs(path, statfs); err != nil {
+			return
+		}
+
+		if statfs.Type != unix.TMPFS_MAGIC {
```

Comment:
I think there also the less common ramfs so I think we should check that here to.